source/routing/next_stop_time.h

```diff
@@ -226,6 +227,16 @@ struct CachedNextStopTime {
                    const JppIdx jpp_idx,
                    const DateTime dt,
                    const bool clockwise) const;
+
+private:
+    struct DtStFromJpp {
```

Comment:
I think it would be nice to have an ASCII art to explain the structure
(and maybe a reminder to explain the range creation in DtStFromJpp::operator[])